crates/bevy_picking_core/src/output.rs

```diff
@@ -335,20 +335,29 @@ pub struct Out;
 pub type PointerDown = PointerEvent<Down>;
 #[derive(Copy, Clone, Eq, PartialEq, Debug, Reflect)]
 /// The inner [`PointerEvent`] type for [`PointerDown`].
-pub struct Down;
+pub struct Down {
+    /// Pointer button pressed to trigger this event.
+    pub button: PointerButton,
+}
 
 /// Fires when a the pointer primary button is released over the `target` entity.
```

Comment:
This comment is incorrect - it fires when *any* pointer button is pressed, as you correctly pointed out. :)

Reply:
Could you also correct the commend on pointerdown please?